### ISSUE TITLE
Updating bgh_smart climate took longer than the scheduled update interval 0:01:00

### ISSUE THREAD

**Issue by JMVS** [mool/home-assistant-bgh-smart#8]
Hola.

Estoy teniendo este mensaje en el registro desde hace varios días (probablemente coincidiendo con la actualización de HA a 2021.6.4, pero no podría asegurarlo)

```
Logger: homeassistant.components.climate
Source: helpers/entity_platform.py:693
Integration: Climatización (documentation, issues)
First occurred: 20:08:27 (7 occurrences)
Last logged: 20:14:27

Updating bgh_smart climate took longer than the scheduled update interval 0:01:00
```

Si puedo conectarme y controlar los equipos desde la aplicación BGH Smart Control.

**Comment by SergioEpstein** [comment]
Indeed, the home-assistant-bgh-smart stopped working, and it happens after the 2021.6.4 update, can someone help?

Thanks a lot

**Comment by dbrachom** [comment]
Hola.. Estoy teniendo el mismo problema, la unica forma de solucionarlo es reiniciando HA, pero muy poco despues vuelve el tema y deja de funcinoar la integracion.

Tienen alguna actualizacion en el tema? gracias cualquier ayuda o idea puede servir..
